AEPIdentity/Tests/IdentityMapTests.swift

```diff
@@ -0,0 +1,274 @@
+//
+// Copyright 2020 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+@testable import AEPIdentity
+import XCTest
+
+class IdentityMapTests: XCTestCase {
+
+    override func setUp() {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        continueAfterFailure = false // fail so nil checks stop execution
+    }
+
+    // MARK: getItemsFor tests
+    func testGetItemsFor() {
+        var identityMap = IdentityMap()
+        identityMap.addItem(namespace: "space", id: "id", authenticationState: XDMAuthenticationState.ambiguous, primary: false)
+
+        let spaceItems = identityMap.getItemsFor(namespace: "space")
+        XCTAssertNotNil(spaceItems)
+        XCTAssertEqual(1, spaceItems?.count)
+        XCTAssertEqual("id", spaceItems?[0].id)
+        XCTAssertEqual("ambiguous", spaceItems?[0].authenticationState?.rawValue)
+        XCTAssertFalse(spaceItems?[0].primary ?? true)
+
+        let unknown = identityMap.getItemsFor(namespace: "unknown")
+        XCTAssertNil(unknown)
+    }
+
+    func testAddItems() {
```

Comment:
Add a few tests for empty namespace and id to help keep the test coverage up. 😄

Reply:
Added :)